assets/lang/sklmessages_en_US.properties

```diff
@@ -62,18 +61,19 @@ editor.close=Close launcher when game starts
 editor.keep=Keep the launcher open
 editor.snap=Snapshots
 editor.snap.tooltip=Allow use of experimental development versions ("snapshots")
-editor.beta=Beta
-editor.beta.tooltip=Allow use of "Beta" Minecraft versions (From 2010–2011)
-editor.alpha=Alpha
-editor.alpha.tooltip=Allow use of "Alpha" Minecraft versions (From 2010)
+editor.historical=Historical
+editor.historical.tooltip=<todo>
 editor.version=Use version
 editor.latest=Use latest version
 editor.compatibility=Compatibility Mode
-editor.compatibility.tooltip=Skins doesn't work with this mode, but some packs are requiring it
-editor.exe=Executable
+editor.compatibility.subtext=This option turn off skins feature,
+editor.compatibility.sublink=read more
```

Comment:
What do you think about changing the comma to period and in the "read more" after this changing the first letter to upper case?
That's how it is presented in Firefox's settings and it looks good:
![firefox-drm-setting](https://user-images.githubusercontent.com/34160838/43218958-46f88c3c-9046-11e8-9273-f5b4f4fb2d56.png)

And I think there should be "turns off", because it's a third person singular in Present Simple.